### PR TITLE
CLDR-13139 Upgrade Tomcat using Ansible

### PR DESCRIPTION
**Pull request by btangmu**
-Revise tomcat-playbook.yml to use robertdebock.tomcat

-Move parts of vars/main.yml to new mysql.yml, tomcat.yml

-Remove roles.yml; use requirements.yml instead in ansible-lint.yml

##### Checklist

- [x] Issue filed: https://unicode-org.atlassian.net/browse/CLDR-13139
- [x] Updated PR title and link in previous line to include Issue number

### CODE REVIEW

File: tools/scripts/ansible/requirements.yml [hunk 1]
```diff
@@ -1,3 +1,13 @@
...
+- src: geerlingguy.mysql
+  version: 3.3.0
+- src: geerlingguy.nginx
+  version: 2.8.0
+- src: derjd.journald
+  version: 0.0.1
+- src: robertdebock.tomcat
+  version: 4.6.2
+# The following are also required for robertdebock.tomcat
+- src: robertdebock.bootstrap
```
Comment (btangmu) [inline tools/scripts/ansible/requirements.yml:10]: can we avoid adding these manually, or test whether they're actually used/needed as dependencies?

Comment (srl295) [reply tools/scripts/ansible/requirements.yml:10]: @btangmu is it a bug in robertdebock.tomcat? maybe file a bug upstream then

**Review by srl295 — APPROVED** [review]
lgtm so far

File: tools/scripts/ansible/requirements.yml (hunk 1 above)
Comment (btangmu) [reply tools/scripts/ansible/requirements.yml:10]: probably not a bug just my lack of understanding

**Comment by btangmu** [comment]
See https://github.com/unicode-org/cldr/pull/854

**Pull request closed without merging**